openapi/src/json-schemas/api/resources/dataset-resource.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Dataset resource",
+    "allOf": [
+        {"$ref": "../json-api-base-resource.json"},
+        {
+            "properties": {
+                "type": {
+                    "const": "dataset"
+                },
+                "id": {
+                    "type": "string",
+                    "const": "0"
+                }
```

Comment:
should we define a more specific `attributes` here and not just "inherit" the generic one?